charts/cbcontainers-agent/cbcontainers-agent-chart/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: cbcontainers-agent-chart
 description: A Helm chart for installing the CBContainers Agent
 type: application
-version: 0.1.0
-appVersion: "3.0.0"
+version: 1.0.0
+appVersion: "2.4.0"
```

Comment:
what is app version from 3.0.0 to 2.4.0?

Reply:
`appVersion` is the version of the app being installed. in our case - the agents.  `3.0.0` is some default you get when you run `helm create`. `2.4.0` is the actual latest version of our agents which we'll install unless someone changes the default

Reply:
`appVersion` is the version of the app being installed. in our case - the agents.  `3.0.0` is some default you get when you run `helm create`. `2.4.0` is the actual latest version of our agents which we'll install unless someone changes the default